### ISSUE TITLE
[Ticket]: Create a better navbar 

### ISSUE THREAD

**Issue by danielschuster-muc** [danielschuster-muc/potter-db#775]
### Project

Website (potterdb.com)

### Describe the ticket you want to create

In creating our [new website](https://beta.potterdb.com) it would be nice to have an improve navbar, which contains some important links.

It should contain:
- Logo (already there)
- Link to our [documentation](https://docs.potterdb.com)
- Link to our [status page](https://status.potterdb.com)
- Hamburger menu, which links to the different DB search pages (e.g. characters, books, movies, etc.)

Make sure to make a branch from `new-website`. 

If you have any questions, feel free to ping me. You can also send some pictures from current progress or make the PR as a draft.
Please the leave a note before starting, so I can assign you to this issue. 

### Acceptance Criteria

- [ ] Has all the required fields / links
- [ ] It is splitted in reasonable components
- [ ] The navbar is mobile-friendly

**Comment by SufyaanKhateeb** [comment]
I can try to solve this.

**Comment by SufyaanKhateeb** [comment]
How should this be designed? Is there a design document?

**Comment by SufyaanKhateeb** [comment]
The status page, is usually linked in the footer. Are you sure you want it in the header?

**Comment by danielschuster-muc** [comment]
> How should this be designed? Is there a design document?

No, there's no design document 😂 
Feel free to think of a layout / design for the navbar yourself.

**Comment by danielschuster-muc** [comment]
> The status page, is usually linked in the footer. Are you sure you want it in the header? And also can we add MUI library?

You probably have a point, I was thinking of just adding a link to the navbar, but def. makes more sense to add it the footer.
No, sorry, we won't add MUI to the new website.

**Comment by danielschuster-muc** [comment]
Closed with #784
